EIPS/eip-7748.md

```diff
@@ -214,6 +214,8 @@ def state_convert(state: State, stride: int):
             # it will return it from the VKT and write it again (i.e: it's a noop).
             # Thus, this operation is correct under both scenarios. That is, it won't
             # write stale data.
+            # Note that if the account is a EOFv1 account, the basic_data code-size and
+            # code-hash will have a constant value as defined in the spec.
```

Comment:
Honestly, I could have avoided this extra comment. But, I just wanted to somehow signal that `set_account` in L220 will actually save the appropriate values in `basic_data` already.

I'm fine with removing this comment if feels too redundant.